saleor/graphql/webhook/subscription_types.py

```diff
@@ -1157,12 +1157,26 @@ def resolve_order(root, info: ResolveInfo):
 
 
 class FulfillmentCreated(SubscriptionObjectType, FulfillmentBase):
+    notify_customer = graphene.Boolean(
+        description=(
+            "If true, send an email notification to the customer." + ADDED_IN_315
+        ),
+        required=True,
+    )
+
     class Meta:
         root_type = "Fulfillment"
         enable_dry_run = True
         interfaces = (Event,)
         description = "Event sent when new fulfillment is created." + ADDED_IN_34
 
+    @staticmethod
+    def resolve_notify_customer(root, _info: ResolveInfo):
+        if hasattr(_info.context, "extra_params"):
+            extra_params = getattr(_info.context, "extra_params")
+            return extra_params.get("notify_customer", True) if extra_params else True
+        return True
+
```

Comment:
I'm not certain about storing the value in in `info.context` and if is there a better approach, please advise

Reply:
Let's look how `gift_card_sent` is handled. In `plugin.py.gift_card_sent`, you pass a dict with objects you need. Then you can access it via `root` in `FulfillmentCreated` subscription type. However, this approach will disable a dry run.